cmake/options.cmake

```diff
@@ -37,6 +37,11 @@ option(MKLDNN_ENABLE_CONCURRENT_EXEC
     CAUTION: enabling this option increases memory consumption"
     OFF) # disabled by default
 
+option(MKLDNN_DISABLE_JIT
```

Comment:
It should be straightforward to detect a non-x86_64 machine using compiler defines. I would suggest using that instead of a CMake switch. What you want to achieve is to not execute any x86-specific code-paths, so the name is misleading too.

Reply:
Good suggestion. Will do that in the next pull request.